strr-base-web/app/composables/useStrrApi.ts

```diff
@@ -16,18 +16,13 @@ export const useStrrApi = () => {
         return undefined
       })
     }
-    // TODO: add type filter in call (need in api first)
-    const resp = await $strrApi<{ registrations: T[], total: number }>('/registrations', {
+    return await $strrApi<{ registrations: T[], total: number }>('/registrations', {
       query: {
         limit,
-        offset
+        offset,
+        registration_type: type
```

Comment:
Where is the `type` coming from?

Reply:
https://github.com/bcgov/STRR/blob/9c1b1ee2fab5244668be1d592b7e9aa914d6c905/strr-base-web/app/composables/useStrrApi.ts#L9